model/ContainerRuntime.php

```diff
@@ -63,11 +62,13 @@ public function getDeliveryContainer($deliveryId)
     public function getRuntime($deliveryId)
     {
         $delivery = $this->getResource($deliveryId);
-        if (!$delivery->exists()) {
+        $runtimeResource = $delivery->getOnePropertyValue($this->getProperty(self::PROPERTY_RUNTIME));
```

Comment:
i am a little bit confused by this
-----------------------------------------
before we did a count on types ` count($this->getTypes())`
and now we do a `getPropertyValues` per specific property
-----------------------------------------
but why do we need to do this at all, 
because i think this should be something that will minimize the interaction

Reply:
Previously we checked if the delivery existed in one query, and then we retrieved the run time property in a second query.
Now I skip the check if the delivery exist and instead trigger the same exception on a missing runtime (deliveries that do not exist do not have a runtime)